extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/tasks/RememberTheMilkTasksExporter.java

```diff
@@ -95,7 +95,9 @@ private ExportResult exportTask(
     for (TaskList taskList : taskLists) {
       if (taskList.taskseries != null) {
         for (TaskSeries taskSeries : taskList.taskseries) {
-          tasks.add(new TaskModel(oldListId, taskSeries.name, taskSeries.notes.toString()));
+          // figure out what to do with notes
+          String notesStr = taskSeries.notes == null ? "" : taskSeries.notes.toString();
```

Comment:
Add TODO. 
Also, do these have any data at all?

Reply:
Done - Not sure if notes will ever have data. They are included in the RememberTheMilkAPI but all the examples don't have anything in the notes tag. 

Not sure if they're populated by other services - needs some more investigation but haven't been able to get a good response from google tasks - filed #327 to track Google Tasks fixes.